common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
+
+BWL_TYPE=@BWL_TYPE@
+BTL_TYPE=@BTL_TYPE@
```

Comment:
There is no need to declare variables for these. Just use `@BWL_TYPE@` directly below.

Reply:
2 reasons I prefer to still declare them:
1. more readable than `[[ "DUMMY" == "DUMMY" ]] && ...`
2. future options to the script to override the BWL type on deployed system (when we won't build natively)

Reply:
> 2 reasons I prefer to still declare them:
> 
>     1. more readable than `[[ "DUMMY" == "DUMMY" ]] && ...`

I disagree. With the variable, it is not immediately apparent what will be done. With the "DUMMY" == "DUMMY" it's immediately clear. You may wonder a bit why something like that would be written, but it's not a big leap for someone familiar with autotools or CMake configure files.


>     2. future options to the script to override the BWL type on deployed system (when we won't build natively)

There are three cases to consider.
1. Development builds on Ubuntu.
2. Development builds on platform.
3. Releases on platform.

The third case will always use the platform's process management, never this script. So I assume you're talking about the second case.

Then, however, whenever you make a build, the script will be regenerated and reinstalled. I admit, due to the CMAKE_MULTIAP_OUTPUT_DIRECTORY hack, it's not actually going to be copied to the target. But if you would instead go through the install directory, the script would always be correct with the bwl/btl that is actually installed on the target.

Yes, it would allow you to do a debug build without BTL and run it on target with the local bus running, but I think that that is so much of a hack that the script doesn't need to support it.

In addition, I definitely *don't* like adding complexity to the code base for some future unproven use case.

Anyway, not a big deal if it stays like it is - it's not adding that much complexity.

Reply:
If you want it for future use of debug, 
Either do `BWL_TYPE=${1-@BWL_TYPE@}`
or
```
if test -z "$BWL_TYPE" 
then
      BWL_TYPE=@BWL_TYPE@
fi
```

Reply:
I took @arnout 's advice eventually.